grails-app/conf/Config.groovy

```diff
@@ -137,6 +137,8 @@ environments {
         grails.serverURL = "http://${localhostAddress}:8080/$appName"
         gogoduck.url = "http://${localhostAddress}:8300/go-go-duck"
         geonetwork.url = "https://catalogue-123.aodn.org.au/geonetwork"
+
+        grails.mail.port = com.icegreen.greenmail.util.ServerSetupTest.SMTP.port
```

Comment:
Before we roll this, remind me to subscribe for SES, so we can use it to send emails and drive this config from chef.

If greenmail is only for dev, then that's all good.